### ISSUE TITLE
Implement changelog fetching

### ISSUE THREAD

**Issue by MichaelFedora** [MalekaiProject/malekaiAPI#8]
Also figure out @TheScree 's data model for it...

**Comment by MichaelFedora** [comment]
```typescript
class ChangeLog {
  changedate: Date;
  data_type: string;
  applies_to: string; // id
  change: string; // description, etc
  attributes: string[]; // tags or something about things changed
  changes: { 
    attribute_name: string;
    attribute_new_value: any;
    attribute_old_value: any;
  }[];
}```

**Comment by MichaelFedora** [comment]
solved via 43d3a02
